.github/workflows/docker.yml

```diff
@@ -15,19 +15,8 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  docker-release:
-    if: github.event_name == 'release'
-    concurrency: docker-release
-    runs-on: ubuntu-latest
-
-    steps:
-      - name: Docker image
-        uses: Icinga/docker-icinga2@master
-        with:
-          dockerhub-token: '${{ secrets.DOCKER_HUB_PERSONAL_TOKEN }}'
-
   docker:
-    if: github.event_name != 'release'
+    concurrency: docker-${{ github.event_name == 'release' && 'release' || github.ref }}
```

Comment:
You can replace the global workflow definition `concurrency` with this (`docker-${{ github.event_name == 'release' && 'release' || github.ref }}`), and this job-specific definition can then be dropped entirely.

Reply:
Absolutely not. The global workflow concurrency cancels same-PR stuff, the one I changed here just waits for other releases to push their Docker images.